ephios/core/signup/flow/builtin/coupled.py

```diff
@@ -0,0 +1,116 @@
+from django import forms
```

Comment:
Yeah I got to search for mentions of `signup_method` and replace it in some more places...

Reply:
It's similar to what we had before, but I agree they could both be a plugin, or both in core. I think because they can be a plugin, it should be a plugin, but also because they provide a lot of the base implementation useful when building custom flows and structures, they should be in core.
I'm more leaning towards the plugin though. What do you think?

Reply:
would prefer both the same way regardless of what we decide on
I experienced multiple instances where people where confused why they couldn't add events and why they needed to enable such basic functionality upfront. Also it rarely makes sense to disable the builtin flows/structures, even if there are custom ones. so i would lean towards core, but the problem probably could be fixed by better pre-activating the builtin flows/methods